pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/readers/DefaultValueColumnReader.java

```diff
@@ -300,6 +301,13 @@ public boolean isNull(int docId) {
     return _defaultValue == null;
   }
 
+  @Override
+  public Object getValue(int docId)
```

Comment:
Added this as its required when we want to do random access on a column but read as an Object (some Pinot interfaces like Partitioner don't have primitive APIs. In such cases, we can read as Object and avoid additional code to read based on data type)